ccdproc/image_collection.py

```diff
@@ -65,7 +65,7 @@ class ImageFileCollection(object):
         value.
     """
     def __init__(self, location=None, keywords=None, info_file=None,
-                 filenames=None):
+                 filenames=None, extension=None):
```

Comment:
Not sure but how about naming it `ext` or `hdu`? At least these are the names for similar parameters in `fits.getheader`, etc. I'd say we should go with `hdu` because that's what is used in `CCDData.read`

Also needs a parameter documentation